assets/dev/scss/frontend/general/_button.scss

```diff
@@ -25,16 +25,31 @@
 	}
 
 	&-icon {
+
+		i {
+			color: var(--button-icon-color, inherit);
+		}
+
 		svg {
 			width: 1em;
 			height: auto;
+			fill: var(--button-icon-color, inherit);
+		}
+
+		svg:hover {
+			fill: var(--button-icon-color, inherit);
 		}
 
 		.e-font-icon-svg {
 			height: 1em;
```

Comment:
```suggestion
			height: 1em;
			font-size: var(--button-icon-size);
```

Reply:
Do you why we have this element here?

It seems to be a duplicate of the SVG styling.

Can you please comment it out for now?

Reply:
Could we delete this? Do we need this?